### ISSUE TITLE
Aruco_Detect pkg import issue

### ISSUE THREAD

**Issue by BrainFog2** [IE-482-582/spring2024#53]
https://github.com/BYUMarsRover/aruco_detect 

Hello Dr. Murray, we are having some issues regarding the cloning of the github mentioned above. This is after I "catkin_make"

The following is what ros replied back:

-- +++ processing catkin package: 'aruco_detect'
-- ==> add_subdirectory(aruco_detect)
-- Using these message generators: gencpp;geneus;genlisp;gennodejs;genpy
-- Could NOT find fiducial_msgs (missing: fiducial_msgs_DIR)
-- Could not find the required component 'fiducial_msgs'. The following CMake error indicates that you either need to install the package with the same name or change your environment so that it can be found.
**CMake Error** at /opt/ros/noetic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):
  Could not find a package configuration file provided by "fiducial_msgs"
  with any of the following names:

    fiducial_msgsConfig.cmake
    fiducial_msgs-config.cmake

  Add the installation prefix of "fiducial_msgs" to CMAKE_PREFIX_PATH or set
  "fiducial_msgs_DIR" to a directory containing one of the above files.  If
  "fiducial_msgs" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  aruco_detect/CMakeLists.txt:5 (find_package)

-- Configuring incomplete, errors occurred!
See also "/home/student/catkin_ws/build/CMakeFiles/CMakeOutput.log".
See also "/home/student/catkin_ws/build/CMakeFiles/CMakeError.log".
make: *** [Makefile:34350: cmake_check_build_system] Error 1
**Invoking "make cmake_check_build_system" failed**

Any help is appreciated!

**Comment by cmurray3** [comment]
It appears that you need to install the fiducial_msgs package.

-chase murray

On May 7, 2024, at 5:58 PM, BrainFog2 ***@***.***> wrote:

﻿

https://github.com/BYUMarsRover/aruco_detect

Hello Dr. Murray, we are having some issues regarding the cloning of the github mentioned above.

The following is what ros replied back:

-- +++ processing catkin package: 'aruco_detect'
-- ==> add_subdirectory(aruco_detect)
-- Using these message generators: gencpp;geneus;genlisp;gennodejs;genpy
-- Could NOT find fiducial_msgs (missing: fiducial_msgs_DIR)
-- Could not find the required component 'fiducial_msgs'. The following CMake error indicates that you either need to install the package with the same name or change your environment so that it can be found.
CMake Error at /opt/ros/noetic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):
Could not find a package configuration file provided by "fiducial_msgs"
with any of the following names:

fiducial_msgsConfig.cmake
fiducial_msgs-config.cmake


Add the installation prefix of "fiducial_msgs" to CMAKE_PREFIX_PATH or set
"fiducial_msgs_DIR" to a directory containing one of the above files. If
"fiducial_msgs" provides a separate development package or SDK, be sure it
has been installed.
Call Stack (most recent call first):
aruco_detect/CMakeLists.txt:5 (find_package)

-- Configuring incomplete, errors occurred!
See also "/home/student/catkin_ws/build/CMakeFiles/CMakeOutput.log".
See also "/home/student/catkin_ws/build/CMakeFiles/CMakeError.log".
make: *** [Makefile:34350: cmake_check_build_system] Error 1
Invoking "make cmake_check_build_system" failed

Any help is appreciated!

**Comment by BrainFog2** [comment]
Yup you are right did not install the full package initially, Thanks
